sysbuild/Kconfig.suit

```diff
@@ -69,6 +69,12 @@ config SUIT_ENVELOPE_NORDIC_TOP_DIRECTORY
 
 endif # SUIT_ENVELOPE
 
+config SUIT_BUILD_RECOVERY
+	bool "Build SUIT recovery firmware"
```

Comment:
Looking at the integration (`hci_ipc` files are assumed to be present), I guess it `depends on SB_CONFIG_NETCORE_HCI_IPC`.

Reply:
Added